python/pyspark/ml/classification.py

```diff
@@ -772,6 +782,10 @@ class GBTClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol
     >>> test1 = sqlContext.createDataFrame([(Vectors.sparse(1, [0], [1.0]),)], ["features"])
     >>> model.transform(test1).head().prediction
     1.0
+    >>> model.totalNumNodes
+    15
+    >>> print(str(model.toDebugString))
+    GBTClassificationModel (uid=...)...with 5 trees...
```

Comment:
The override for `trees` in `GBTClassificationModel` to return a list of `DecisionTreeRegressionModel` is missing.